Vagrantfile

```diff
@@ -0,0 +1,38 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision "shell", :privileged => false, inline: <<-SHELL
+    sudo apt-get update
+
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+    git clone https://github.com/pyenv/pyenv.git /home/vagrant/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> /home/vagrant/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> /home/vagrant/.profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> /home/vagrant/.profile
+
+    exec "$SHELL"
+  SHELL
+  
+  config.vm.provision "shell", :privileged => false, inline: <<-SHELL
+    pyenv install 3.8.5
+    pyenv global 3.8.5
+
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+    source $HOME/.poetry/env
```

Comment:
Are you expecting this line to do anything?

That env file just adds poetry to the PATH but this is the end of the script.
And you don't need to source .poetry/env because installing poetry already appends the same PATH update command to the .profile